src/components/AccountablePeopleList.svelte

```diff
@@ -0,0 +1,104 @@
+<script lang="ts">
+import type { PolicyDependent } from 'data/dependents'
+import type { PolicyMember } from 'data/types/policy-members'
+import { IconButton, Tooltip } from '@silintl/ui-components'
+import { createEventDispatcher } from 'svelte'
+
+export let policyMembers: PolicyMember[] = []
+export let dependents: PolicyDependent[] = []
+export let isHouseholdPolicy = false
+export let userID: string
+
+$: isYou = (member: PolicyMember) => userID === member.id
+
+const dispatch = createEventDispatcher()
```

Comment:
Do you want to add typing here? (optional)
```suggestion
const dispatch = createEventDispatcher()
const dispatch = createEventDispatcher<{ deleteMember: PolicyMember; editDependent: PolicyDependent; editProfile: void }>()
```
I'm not sure of the syntax for the `void` one, but that's my guess.